src/Forms/ElementalAreaField.php

```diff
@@ -247,6 +248,9 @@ public function saveInto(DataObjectInterface $dataObject)
 
             $element->updateFromFormData($data);
             $element->write();
+            $elements[] = $element;
         }
+
+        $this->extend('onElementsSaved', $elements);
```

Comment:
does this follow convention for extension hook naming? should it be `extend('updateSaveInto')` ?

Reply:
Have updated to `onSaveInto` with the `$elements`, `$dataObject` and `$elementData` parameters.